src/elements/ops.rs

```diff
@@ -311,6 +311,184 @@ impl Opcode {
             _ => false,
         }
     }
+
+    /// The mnemonic for this opcode.
+    pub fn mnemonic(&self) -> &str {
+        match *self {
```

Comment:
you can also add `use elements::Opcode::*` and predicate `Opcode::` in every line